### PR TITLE
Fix/add author reorder option

### PR DESCRIPTION
**Pull request by melisabok**
Submission Revision Stage: add a new field to select how the author list is editable.

### CODE REVIEW

File: openreview/venue_request/venue_request.py [hunk 1]
```diff
@@ -377,15 +377,25 @@ def setup_submission_revision_stage(self):
                 ],
                 'default': 'Enable revision for all submissions',
                 'required': True,
-                'order': 38
+                'order': 4
+            },
+            'submission_author_edition': {
+                'description': 'Choose how to edit the author list',
```
Comment (celestemartinez) [inline openreview/venue_request/venue_request.py:383]:
```suggestion
                'description': 'Choose how authors may edit the author list',
```

**Review by celestemartinez — APPROVED** [review]

File: openreview/venue_request/venue_request.py [hunk 2]
```diff
@@ -377,15 +377,25 @@ def setup_submission_revision_stage(self):
...
+                'value-radio': [
+                    'Allow add and remove authors',
```
Comment (celestemartinez) [inline openreview/venue_request/venue_request.py:385]:
```suggestion
                    'Allow addition and removal of authors',
```

File: openreview/venue_request/venue_request.py [hunk 3]
```diff
@@ -377,15 +377,25 @@ def setup_submission_revision_stage(self):
...
+                    'Allow order of the authors only'
```
Comment (celestemartinez) [inline openreview/venue_request/venue_request.py:386]:
```suggestion
                    'Allow reorder of existing authors only'
```

File: openreview/venue_request/venue_request.py (hunk 1 above)
Comment (melisabok) [reply openreview/venue_request/venue_request.py]: Done

File: openreview/venue_request/venue_request.py (hunk 2 above)
Comment (melisabok) [reply openreview/venue_request/venue_request.py]: Done

File: openreview/venue_request/venue_request.py (hunk 3 above)
Comment (melisabok) [reply openreview/venue_request/venue_request.py]: Done

**Pull request merged**